model/src/commonMain/kotlin/io/github/droidkaigi/feeder/FeedContents.kt

```diff
@@ -50,17 +48,13 @@ fun fakeFeedContents(): FeedContents {
                 media = Media.Medium,
                 title = MultiLangText(
                     jaTitle = "DroidKaigi 2020でのCodelabsについて",
-                    enTitle = "DroidKaigi 2020 Codelabs",
+                    enTitle = "DroidKaigi 2020でのCodelabsについて"
                 ),
                 summary = MultiLangText(
-                    jaTitle = "DroidKaigiスタッフの@tomoya0x00です。DroidKaigi " +
-                        "2020の開催日が近づいてきましたね。この記事では、DroidKaigi 2020 Day.1に実施する " +
-                        "DroidKaigiオリジナルCodelabsについてご紹介します。",
-                    enTitle = "This is tomoya0x00 from DroidKaigi Committee. DroidKaigi 2020 is " +
-                        "approaching!This post is about our original codelab that you can try on…",
-                ),
-                link = "https://medium.com/droidkaigi/droidkaigi-2020%E3%81%A7%E3%81%AE" +
-                    "codelabs%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6-4a677d7456c6",
+                    jaTitle = "DroidKaigiスタッフの@tomoya0x00です。DroidKaigi 2020の開催日が近づいてきましたね。この記事では、DroidKaigi 2020 Day.1に実施する DroidKaigiオリジナルCodelabsについてご紹介します。",
```

Comment:
Can you create FakeFeedContents.kt and exclude it? 🙇 
https://stackoverflow.com/a/46115703/4339442

```
java.lang.AssertionError: Error on line: 54, column: 1
Exceeded max line length (100)
```

Reply:
added rules and create file!

https://github.com/DroidKaigi/conference-app-2021/pull/216/commits/c5094201ee213b4f272dd0751efc838b8a4a2eb3